bot/cogs/eval_cog.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, bot):
     @ext.short_help('Runs arbitrary python code in discord')
     @ext.example('eval print("hello world")')
     async def eval(self, ctx, *, code= None) -> None:
+        code = code.replace('```python', '')
```

Comment:
you can also specify python syntax like \```py    ``` so we should add that as well. Other then that yes plz, this has been on my list for ages

Reply:
Done